cli/morphir-elm-develop.js

```diff
@@ -19,7 +19,7 @@ program
     "Start up a web server and expose developer tools through a web UI"
   )
   .option("-p, --port <port>", "Port to bind the web server to.", "3000")
-  .option("-o, --host <host>", "Host to bind the web server to.", "0.0.0.0")
+  .option("-o, --host <host>", "Host to bind the web server to.", "localhost")
```

Comment:
This is appropriate for running Morphir Web locally, but it will not work when deployed as a web endpoint. Can you please confirm that we are overriding the default to "0.0.0.0" while dockerizing?

Reply:
Yes this is confirmed. The 0.0.0.0 is not used for in for containers. Container names are used.